xml/favicons.xml

```diff
@@ -2227,4 +2227,12 @@
     <param pos="0" name="service.product" value="Castopod"/>
   </fingerprint>
 
+  <fingerprint pattern="^(?:9e8ba456e7e39ea364cc538959813719|86157069c9574d4c75b907e614d6a521)$">
+    <description>Vaultwarden - unofficial Bitwarden server</description>
```

Comment:
Can we call it "unofficial Bitwarden compatible server" because that's what they call it? It's not really a pirate Bitwarden server, and they go to some pains to disambiguate on that for trademarks etc.

Reply:
I ended up grabbing that phrase from [their code](https://github.com/search?q=repo%3Adani-garcia%2Fvaultwarden+unofficial&type=code), but I agree the phrase from their repo description would be good.
> unofficial Bitwarden&reg; server
> *unofficial* Bitwarden implementation